package.json

```diff
@@ -38,6 +38,7 @@
     "url": "https://github.com/jenkinsci/jenkins-design-language.git"
   },
   "dependencies": {
+    "lodash.debounce": "4.0.8",
     "moment": "2.15.1",
     "moment-duration-format": "1.3.0",
     "react": "15.3.2",
```

Comment:
Writing my own `debounce` seemed silly but I can see maybe just pulling this directly into our repo to cut down on the dependency headaches.